src/configuration/rawApiConfiguration.ts

```diff
@@ -0,0 +1,41 @@
+export interface RawApiConfiguration {
+    deployment: DeploymentDefinition;
+}
+
+export interface DeploymentDefinition {
+    codesConverter: [CodeConverter];
```

Comment:
I want to define a type that represents the result of xml parsing.
So I ave no choice. In the xml, the Deployment tag contains a CodeConverter tag...
I agree with you. The naming is not very good. It could be improved on the xml side (not api side)